org.eclipse.transformer/src/main/java/org/eclipse/transformer/action/impl/ClassActionImpl.java

```diff
@@ -377,7 +377,7 @@ public ByteData apply(String inputName, byte[] inputBytes, int inputLength) thro
 		}
 		while (fields.hasNext()) {
 			FieldInfo inputField = fields.next();
-			FieldInfo outputField = transform(inputField, FieldInfo::new, SignatureType.FIELD);
+			FieldInfo outputField = transform(inputField, FieldInfo::new, SignatureType.FIELD, inputName);
```

Comment:
All of these 'inputName' parameters are ugly, but that's not the fault of this update, but rather of the need to keep actions stateless.  There is room to refactor how the actions manage state, which is more than should be put into this pull request.